Web/Scotty.hs

```diff
@@ -89,6 +90,11 @@ defaultHandler = Trans.defaultHandler
 middleware :: Middleware -> ScottyM ()
 middleware = Trans.middleware
 
+-- | Set global request size limit. Requests with size exceeding the limit will not be
+-- processed and an HTTP error code will be returned to the client. 
+maxRequestSize :: Kilobytes -> ScottyM ()
```

Comment:
nitpick: IIUC this limits the length of the request body, not the entire request. Perhaps `maxRequestBodySize` is a better name?

Reply:
hey @fumieval, you are absolutely right and that's a valid point. I updated the naming.

Please if you spot anything else let me know so I can improve the PR. Thanks! :)